src/gui/screens/transaction.py

```diff
@@ -1,8 +1,49 @@
 import lvgl as lv
+import utime
+import platform
 from .prompt import Prompt
 from ..common import add_label, format_addr
 from ..decorators import on_release
 
+# The conv_time() function converts a timestamp measured in seconds from 1970-01-01 00:00:00 UTC to
+# humand-readable parameters (year, month, day, hour, minute, second, second, weekday, yeardate) in UTC.
+# "Time Epoch: Unix port uses standard for POSIX systems epoch of 1970-01-01 00:00:00 UTC.
+# However, embedded ports use epoch of 2000-01-01 00:00:00 UTC."
+# (Source: https://micropython.readthedocs.io/en/latest/library/utime.html)
+# Simulator MicroPython case:
+#   utime.mktime() does not exist. utime.localtime() gives result with both timezone offset and DST offset.
+#   To remove the timezone offset, we calculate the offset of EPOCH ZERO timestamp (1970-01-01 00:00:00 UTC),
+#   substract it from the timestamp, and recall utime.localtime() again.
+#   To remove the DST offset, we usually only need to check the dst_offset in the result (in hours), subtract it
+#   and call utime.localtime() again. However, in one case (DST start on March) when the local clocks jump forward,
+#   there is an hour when we don't need to apply the shift - and we correct it manually.
+# Embedded MicroPython case:
+#   utime.gmtime() and utime.localtime() are the same function (in some implementation only utime.localtime()
+#   exists, but does not add timezone/dst offsets). However, timestamp zero is not 1970-01-01 00:00:00 UTC,
+#   but 2000-01-01 00:00:00 UTC. Therefore we reduce the fixed difference from the timestamp before execution.
```

Comment:
Good comment explanation. In order to make it better maintainable, i'd move the if-clause within the conv-function and also move the whole function to `src/helpers.py` ?!
That strange difference seems to make this function unavaidable.